modules/custom/express_site_metrics/express_site_metrics.info

```diff
@@ -0,0 +1,11 @@
+name = Express Site Metrics
+description = Provides an endpoint for Prometheus to access and record an CU Web Express websites metrics
+core = 7.x
+
+files[] = classes/express_site_metrics_inspector.php
+files[] = classes/express_site_metrics_token_handler.php
+files[] = classes/express_site_metrics_variable_controller.php
+
+dependencies[] = atlas
+dependencies[] = atlas_statistics
+dependencies[] = cu_users
```

Comment:
This module needs to be enabled via an update hook, probably in express core

Reply:
Do you mean cu_core, right? Does that also need a change in this .info file. Or just `cu_core_install()`?

Reply:
I think you need two things, one is to declare a dependency in `cu_core.info` so that is on for all new sites and additionally and update hook in `cu_core.install` so that it is on for all existing sites.